repos/spack_repo/builtin/packages/visit/package.py

```diff
@@ -196,6 +196,7 @@ class Visit(CMakePackage):
     depends_on("mfem@4.4:", when="+mfem")
     depends_on("mfem+shared+exceptions+fms+conduit", when="+mfem")
     depends_on("libfms@0.2:", when="+mfem")
+    conflicts("^mfem build_system=cmake")
```

Comment:
```suggestion
    depends_on("mfem+shared+exceptions+fms+conduit build_system=cmake", when="+mfem")
    depends_on("libfms@0.2:", when="+mfem")
```

Perhaps this is clearer?

Reply:
I think this changes the logic since it is forcing the package to use the `CMake` build rather than forcing it to not use the `CMake` build.

Reply:
gah yes. sorry this would be better, or however it is you would express dependency logic on the GNU Make based build:

```suggestion
    depends_on("mfem+shared+exceptions+fms+conduit build_system=make", when="+mfem")
    depends_on("libfms@0.2:", when="+mfem")
```